bootstrap/ci/main.tf

```diff
@@ -12,14 +12,27 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 
+module "naming" {
+  source      = "../shared/naming"
+  id          = var.id
```

Comment:
for me a bare `id` variable is pretty confusing
(feel free to ignore if it is for others)

Reply:
What's confusing about it? Any alternatives? (I changed it from suffix as it's not a suffix, the suffix is generated from this and the environment

`flowehr_id`?

Reply:
> flowehr_id?

💯